nomenclature/codelist.py

```diff
@@ -624,3 +624,62 @@ def filter(self, hierarchy: str) -> "RegionCodeList":
                 "Use `RegionCodeList.hierarchy` method for available items."
             )
             raise ValueError(msg)
+
+
+class MetaCodeList(CodeList):
+    """A subclass of CodeList specified for MetaCodes
+
+    Attributes
+    ----------
+    name : str
+        Name of the MetaCodeList
+    mapping : dict
+        Dictionary of `MetaCode` objects
+
+    """
+
+    # class variable
+    code_basis: ClassVar = MetaCode
+    validation_schema: ClassVar[str] = "meta indicators"
+
+    @classmethod
+    def from_yaml_files(cls, name: str, path: Path, file_glob_pattern: str = "**/*"):
```

Comment:
Why not keep the `from_directory()` convention here? I think we should keep the meta-indicators in a "meta" folder for consistency.

Reply:
I second that one it should definitely be called `from_directory()` otherwise `DataStructureDefinition` cannot use it.
Probably should implement a `MetaCodeList` class that enforces that ...

Reply:
Looking at this again, I don't think the method is actually needed at all. Since `MetaCodeList` inherits from `CodeList` and `CodeList` implements `from_directory` we should be good.